os/pm/pm_timedsuspend.c

```diff
@@ -124,7 +124,7 @@ int pm_timedsuspend(int domain_id, unsigned int milliseconds)
 	/* If delay is zero then cancel the timer (PM Policy) */
 	if (delay == 0) {
 		if (wdog) {
-			timer_timeout(NULL, domain_id);
```

Comment:
Looks like timer_timeout is taking int as argument, I am not sure why we are passing NULL instead of 0 here.

https://github.com/Samsung/TizenRT/blob/1d5428977070e9c9db3fe37593086e37536bedda/os/pm/pm_timedsuspend.c#L80-L89